pkgs/development/libraries/graphene-hardened-malloc/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "graphene-hardened-malloc";
-  version = "2";
+  version = "QQ2A.200405.005.2020.04.07.10";
```

Comment:
Has the versioning policy changed? Integer releases used to be for standalone releases, while the others were more closely tied to grapheneos.

Reply:
Seems you're right, the tagged commit is https://github.com/GrapheneOS/hardened_malloc/commit/87296f98547901bfcf431eaf8401c6f312192e2a which shouldn't apply to NixOS. I'll drop this commit before merge.